distribution/scripts/jmeter/generate-payload.sh

```diff
@@ -60,5 +60,8 @@ fi
 
 for s in ${payloads[*]}; do
     echo "Generating ${s}B file"
+    var_path=`pwd`
```

Comment:
What is the directory path (value of `var_path`) we are at when generating the payloads?

Reply:
Its home directory, changed it .